packages/snap/test/unit/wallet.mock.ts

```diff
@@ -0,0 +1,31 @@
+import { Asset, Wallet } from '../../src/interfaces';
+import sinon from 'sinon';
+
+export class WalletMock implements Wallet {
+  public registerRpcMessageHandler = sinon.stub();
+  public send = sinon.stub();
+  public request = sinon.stub();
+
+  public reset(): void {
+    this.registerRpcMessageHandler.reset();
+    this.send.reset();
+    this.request.reset();
+  }
+}
+
+export const getWalletMock = (): WalletMock => {
+  // eslint-disable-next-line
+  // @ts-ignore
+  return global.snap as WalletMock;
+};
+
+export function executeAssetOperation(
+  asset: Partial<Asset>,
+  method: 'update' | 'add' | 'remove'
+): Promise<Asset> {
+  const snap = getWalletMock();
+  return snap.send({
+    method: 'wallet_manageAssets',
+    params: [method, asset]
+  }) as Promise<Asset>;
+}
```

Comment:
can you explain a point of this code?
As is not used anywhere, only inside `packages/snap/test/unit/asset/action.test.ts` but that dose not make sense to me

Reply:
It was moved from src/asset The method was used only inside the tests, so I moved it to the tests folder. But I can't see that our snap could work with assets. Probably, it's some legacy code or undeveloped try to bring assets management. If you don't have a scenario where it could be used, we could remove the code.

Reply:
Yeah seems like and should be cleaned up

Reply:
removed